tests/Services/AuthServiceTest.php

```diff
@@ -14,7 +14,7 @@ public function testGetCurrentUserWithAccessToken()
         $tokenService = new TokenService([
             'client_id' => env('CLIENT_ID'),
             'client_secret' => env('CLIENT_SECRET'),
-            'base_url' => env('LI_BASE_URL', 'http://localhost:33001'),
+            'base_url' => env('LI_BASE_URL', 'http://localhost:3000'),
```

Comment:
Q: Why are we changing the ports here?

Reply:
I was getting an error when running the unit tests stating that `localhost:33001/strategy` can not be found in the snapshots. So I changed it to use the heimdal port. Now I'm not sure if that's the right approach but that's the reason behind it. Same reason in ServiceTestCase.